design/workitemtype.go

```diff
@@ -21,9 +21,15 @@ var fieldDefinition = a.Type("fieldDefinition", func() {
 	a.Description("A fieldDescription aggregates a fieldType and additional field metadata")
```

Comment:
Shouldn't this be -> 
`a.Description("A fieldDefinition aggregates a fieldType and additional field metadata")`